aclk/README.md

```diff
@@ -61,12 +61,36 @@ claiming](/claim/README.md#claim-through-a-proxy).
 You can configure following keys in the `netdata.conf` section `[cloud]`:
 ```
 [cloud]
-    statistics = yes
-    query thread count = 2
+  statistics = yes
+  query thread count = 2
+  aclk implementation = legacy
 ```
 
 - `statistics` enables/disables ACLK related statistics and their charts. You can disable this to save some space in the database and slightly reduce memory usage of Netdata Agent.
 - `query thread count` specifies the number of threads to process cloud queries. Increasing this setting is useful for nodes with many children (streaming), which can expect to handle more queries (and/or more complicated queries).
+- `aclk implementation` - see [ACLK implementation](#aclk-implementation) section
+
+## ACLK implementation
+
+Currently we are in process of switching ACLK to brand new technical stack called ACLK-NG. To choose implementation your agent should use you can change the `aclk implementation` setting in your `netdata.conf` (accepted values `ng` or `legacy`).
+
+Before changing this value check the desired implementation is available in your agent (determined at build time) by running `netdata -W buildinfo`. Following lines indicate which ACLK implementations are available in your agent:
+
+```
+Features:
+    ACLK Next Generation:    YES
+    ACLK Legacy:             YES
+```
+
+To verify which ACLK implementation your agent uses you can visit `/api/v1/info` endpoint on your local dashboard and check the `aclk-implementation` key.
+
+Please note new Cloud features will be implemented on top of ACLK-NG from this point on. ACLK Legacy is therefore kept as fallback in case some users have issues with ACLK-NG or need to use features which are not yet available in ACLK-NG *(like IPv6 support and SOCKS proxy)*.
```

Comment:
```suggestion
New Netdata Cloud features will be implemented on top of ACLK-NG from this point on. ACLK Legacy is therefore kept as a fallback in case some users have issues with ACLK-NG or need to use features which are not yet available in ACLK-NG *(like IPv6 support and SOCKS proxy)*.
```